### PR TITLE
Translation and modifications in extractor.bat

### PR DESCRIPTION
**Pull request by pangolp**
## Changes Proposed:
-  Create the mmaps and vmaps directories if they do not exist, as well as add some messages to tell the user to press a key to continue, in the case of vmaps, and mmaps
-  The extractor file is created in Spanish.

## Issues Addressed:

- Closes 

## Tests Performed:

- Windows 10

## How to Test the Changes:

1. Move the extractor to where the game client is located, along with the executables, and use it.
2. You can use either the original extractor or the Spanish extractor, both work in the same way.
3. If you did not create the mmaps and vmaps folders before running the script, the script will create them for you.

## Screenshot
extractor

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: apps/extractor/extractor.bat [hunk 1]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
 start /b /w vmap4extractor.exe
+if not exist vmaps md vmaps
 start /b /w vmap4assembler.exe Buildings vmaps
 rmdir Buildings /s /q
+cls
 GOTO MENU
 
 :MMAPS
 ECHO This may take a few hours to complete. Please be patient.
+ECHO Press a key to continue.
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:50]: You dont need this. PAUSE will prompt this message by itself

File: apps/extractor/extractor.bat [hunk 2]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
 
 :MAPS
 start /b /w mapextractor.exe
+cls
 GOTO MENU
 
 :VMAPS
 start /b /w vmap4extractor.exe
+if not exist vmaps md vmaps
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:42]: This does nothing.

use something like
```batch
if exist maps\ (
    echo folder found.
) else (
    echo creating folder "maps".
    mkdir "maps"
)
```

File: apps/extractor/extractor.bat [hunk 3]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
 start /b /w vmap4assembler.exe Buildings vmaps
 rmdir Buildings /s /q
+cls
 GOTO MENU
 
 :MMAPS
 ECHO This may take a few hours to complete. Please be patient.
+ECHO Press a key to continue.
 PAUSE
+if not exist mmaps md mmaps
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:52]: This does nothing.

use something like
```batch
if exist maps\ (
    echo folder found.
) else (
    echo creating folder "maps".
    mkdir "maps"
)
```

File: apps/extractor/extractor.bat [hunk 4]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
 start /b /w mmaps_generator.exe
+cls
 GOTO MENU
 
 :ALL
 ECHO This may take a few hours to complete. Please be patient.
+ECHO Press a key to continue.
 PAUSE
+if not exist vmaps md vmaps
+if not exist mmaps md mmaps
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:62]: This does nothing.

use something like
```batch
if exist maps\ (
    echo folder found.
) else (
    echo creating folder "maps".
    mkdir "maps"
)
```

File: apps/extractor/extractor.bat [hunk 5]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
+cls
 GOTO MENU
 
 :MMAPS
 ECHO This may take a few hours to complete. Please be patient.
+ECHO Press a key to continue.
 PAUSE
+if not exist mmaps md mmaps
 start /b /w mmaps_generator.exe
+cls
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:54]: dont run cls. If the extractor fails, we want to know why

File: apps/extractor/extractor.bat [hunk 6]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
+ECHO Press a key to continue.
 PAUSE
+if not exist vmaps md vmaps
+if not exist mmaps md mmaps
 start /b /w mapextractor.exe
 start /b /w vmap4extractor.exe
 start /b /w vmap4assembler.exe Buildings vmaps
 rmdir Buildings /s /q
 start /b /w mmaps_generator.exe
+cls
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:68]: dont run cls. If the extractor fails, we want to know why

File: apps/extractor/extractor.bat [hunk 7]
```diff
@@ -34,26 +34,36 @@ IF %M%==5 GOTO :EOF
...
+ECHO Press a key to continue.
 PAUSE
+if not exist mmaps md mmaps
 start /b /w mmaps_generator.exe
+cls
 GOTO MENU
 
 :ALL
 ECHO This may take a few hours to complete. Please be patient.
+ECHO Press a key to continue.
```
Comment (Kitzunu) [inline apps/extractor/extractor.bat:59]: You dont need this. PAUSE will prompt this message by itself

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Helias** [comment]
???

**Comment by pangolp** [comment]
I got the wrong PR hahaha

**Comment by pangolp** [comment]
Ready, I applied all the changes, although I did not test them. I did test them the old way and it works without problems. With these new changes, I really don't know if it will work. I ask you to tell me if it works @Kitzunu

**Comment by Kitzunu** [comment]
if the point is to create the folders, then yes this will work.

**Comment by pangolp** [comment]
Yes, the goal is to create the folders, in case they don't exist, since in the wiki, it mentions that they should be created, so basically the idea is to create them, in case people don't do it.

**Comment by Kitzunu** [comment]
then it works =)

**Comment by pangolp** [comment]
If you believe that all requested changes have been corrected, please change your change request, so that others can approve it. Thank you. If not, let me know what other changes I need to make. @Kitzunu

**Review by Kitzunu — APPROVED** [review]

**Comment by pangolp** [comment]
@Kitzunu thanks.

**Comment by FrancescoBorzi** [comment]
is this ready to be merged? if yes apply [To Be Merged] please...

**Pull request merged**